src/main/java/Llama.java

```diff
@@ -0,0 +1,65 @@
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+import commands.Command;
+import data.Storage;
+import data.TaskList;
+import exceptions.LlamaException;
+import parser.Parser;
+import ui.Ui;
+
+/**
+ * Main class for program. Responsible for starting and ending the program
+ */
+public class Llama {
+    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+
+    private Storage storage;
+    private Ui ui;
+    private TaskList taskList;
+
+    /**
+     * Constructor for Llama
+     */
+    public Llama() {
+        this.ui = new Ui();
+        this.storage = new Storage();
+        try {
+            this.taskList = storage.load();
+        } catch (IOException e) {
+            ui.displayString(e.getMessage()); // TODO: Make own exception for custom message
+        }
+    }
+
+    /**
+     * Method to start running the program
```

Comment:
Shouldn't the first sentence of a method's JavaDoc start in the form of a verb? (According to [this](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=In%20method%20header%20comments%2C%20the%20first%20sentence%20should%20start%20in%20the%20form%20Returns%20...%2C%20Sends%20...%2C%20Adds%20...%20etc.%20(not%20Return%20or%20Returning%20etc.)))

I also noticed this in other JavaDoc comments.